tests/integration/README.md

```diff
@@ -79,68 +80,52 @@ echo 'monitord.rotate_log=0' >> $wazuh_path/etc/local_internal_options.conf
 choco install jq
 ```
 
-- Install Python dependencies
+- Install integration tests dependencies
 
 ```shell script
-pip install pytest freezegun jsonschema pyyaml==5.4 psutil paramiko distro pywin32 pypiwin32 wmi pandas==0.25.3 pytest-html==2.0.1 numpydoc==0.9.2
+cd wazuh-qa
+python -m pip install -r requirements.txt
 ```
 
-- Change `time-reconnect` from `C:\Program Files (x86)\ossec-agent\ossec.conf`
```

Comment:
More internal options to take into consideration.

Reply:
Done 15215b391ba5d18eae8923b418137fd9f649b541